Photon-Tinker/Mesh/MeshSetupFlowUIManager.swift

```diff
@@ -976,6 +1008,30 @@ class MeshSetupFlowUIManager : UIViewController, Storyboardable, MeshSetupFlowMa
         }
     }
 
+    @IBAction func backTapped(_ sender: UIButton) {
+        //resume previous VC
+        let vcs = self.embededNavigationController.viewControllers
+        log("Back tapped: \(vcs)")
+
+        if (vcs.last! as! MeshSetupViewController).viewControllerIsBusy {
+            log("viewController is busy, not backing")
+            //view controller cannot be backed from at this moment
+            return
+        }
+
+        if vcs.count > 1 {
+            (vcs[vcs.count-2] as! MeshSetupViewController).resume(animated: false)
+        }
+
+        if (vcs.last! as! MeshSetupViewController).rewindFlowOnBack {
```

Comment:
Why is it necessary to handle certain back events differently? Is it when you have several steps in the same flow so that in case you are in a later step, you just pop the view controller but when you are in the first step, you actually rewind the flow?